packages/legacy/core/App/screens/ListCredentials.tsx

```diff
@@ -36,9 +40,13 @@ const ListCredentials: React.FC = () => {
   const { ColorPallet } = useTheme()
   const { start } = useTour()
   const screenIsFocused = useIsFocused()
-  let credentials = [
+  const { openIdState } = useOpenIDCredentials()
+  const { w3cCredentialRecords } = openIdState
```

Comment:
```suggestion
  const { openIdState: { w3cCredentialRecords } } = useOpenIDCredentials()
```

Just an alternative way, don't have to use this if you don't want to

Reply:
great idea, much cleaner